login.md

```diff
@@ -1,27 +1,41 @@
 
 ## Logging in to our Class Server 
-Sutdents will recieve an account on the server AP, which is a server instance running on Google Cloud Platform. Think of this as signing onto a remote server where you will do all your work instead of working on and saving files to your local machine. You will access the class server using SSH, which allows a secure terminal session with our class server.
+Students will recieve an account on the class server named, AP, which is a server instance running on Google Cloud Platform. Your account allows you access to a remote server where you will do all your work and save all your files. 
 
-For those of you using macOS, we recommend using its [Terminal](https://support.apple.com/guide/terminal/welcome/mac).
+For those of you using macOS, we recommend using its [Terminal](https://support.apple.com/guide/terminal/welcome/mac). If you are using a Windows machine , we recommend using [cygwin](https://www.cygwin.com/), [PuTTY](https://www.putty.org/), [MobaX](https://mobaxterm.mobatek.net/), just to name a few. 
 
-On Windows, we recommend using [cygwin](https://www.cygwin.com/), [PuTTY](https://www.putty.org/), [MobaX](https://mobaxterm.mobatek.net/), just to name a few.
-
-Type this into the command-line to establish your connection to our server: 
+Now, to log onto our remote class server, you will need to establish use SSH (Secure SHell).Type this into the command-line to establish your connection to our server: 
 
 ```yaml
 ssh YOUR_UNI@ap.cs.columbia.edu
 ```
-You should be prompted to input your password which you should have recieved in an email. Note, it might seem like you are typing in nothing when typing in your password. 
+You should be prompted to input your password which you should have recieved in an email. Don't worry, it will seem like you are not typing anything but continue and press ENTER when done. If you followed these steps closely, you should see this message appear: 
+```yaml
+                  __                             __      
+   _      _____  / /________  ____ ___  ___     / /_____ 
+  | | /| / / _ \/ / ___/ __ \/ __ `__ \/ _ \   / __/ __ \
+  | |/ |/ /  __/ / /__/ /_/ / / / / / /  __/  / /_/ /_/ /
+  |__/|__/\___/_/\___/\____/_/ /_/ /_/\___/   \__/\____/ 
+                                                       
+                     :::     :::::::::  
+                   :+: :+:   :+:    :+: 
+                  +:+   +:+  +:+    +:+ 
+                 +#++:++#++: +#++:++#+  
+                 +#+     +#+ +#+        
+                 #+#     #+# #+#        
+                 ###     ### ###        
 
-## Writing Code with a Text Editor
+```
+
+## Text Editors
 In this course, you will be writing code using a command line text editor on the class server. In previous semesters, most students and professors recommend Vim. But you can do your own research and use any text-editor that you feel comfortable with (for example, Emacs). 
```

Comment:
I think we have to be careful here about them "doing their own research" cause we don't want them to arrive at the conclusion that they can use VSCode through SSH, so you might want to rewrite this to narrow the options.

Reply:
@xurxo-riesco Dr. B said students can use whatever they want. But I agree that we should refine their options to at most like 3